.circleci/config.yml

```diff
@@ -29,11 +29,15 @@ jobs:
             ccache --max-size=1G
       - run:
           name: cmake
-          command: cmake -DCOVERAGE=ON -H$IROHA_HOME -B$IROHA_BUILD
+          command: cmake -DCOVERAGE=ON -H$IROHA_HOME -B$IROHA_BUILD -DPACKAGE_DEB=ON -DCMAKE_BUILD_TYPE=Release
```

Comment:
>-DCMAKE_BUILD_TYPE=Release

I just realized, coverage info will not be collected, because of Release build type. We have to do 2 builds, one for debug and one for release.